### ISSUE TITLE
Mouse click on qualified items

### ISSUE THREAD

**Issue by UffeJakobsen** [otsaloma/catapult#30]
if you type in something in to the search field and do not press enter on the qualified items but decide to use the mouse to select one - that does not work - for some strange resaon it shows the internal ":" options when clicking with the mouse

**Comment by otsaloma** [comment]
Yeah, I guess that could be fixed. As Catapult is inherently a keyboard-driven app, I haven't bothered to hook up mouse clicks.

The reason for triggering the internal command list is that since the existence of those is quite undiscoverable, I have wanted to provide multiple routes to somehow discover them, especially the preferences dialog. Some launchers have a clicable gear button or a right-clickable window margin, so users might look for that. That being connected to the search results too is just by accident, it should better be just around search query input field.
